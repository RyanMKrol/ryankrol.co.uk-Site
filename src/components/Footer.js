import React from 'react';

import './Footer.css';

/**
 * Class representing a UI component for the footer
 *
 * @returns {React.Component} The JSX representing the component
 */
function Footer() {
  return <div className="Footer" />;
}

export default Footer;
