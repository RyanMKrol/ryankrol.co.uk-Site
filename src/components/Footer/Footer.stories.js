import React from 'react';
import Footer from './Footer';

export default {
  title: 'Components/Footer',
};

/**
 * default story
 *
 * @returns {string} JSX for the component I want to display
 */
export const normal = () => (
  <div style={{ 'background-color': 'black' }}>
    <Footer />
  </div>
);
