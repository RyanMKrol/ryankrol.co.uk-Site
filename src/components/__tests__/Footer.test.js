import React from 'react'
import { shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'

import Footer from './../Footer'

it('renders router only when rendered', () => {
    const footer = shallow(<Footer />)
    expect(toJson(footer)).toMatchSnapshot()
})

it('does not crash when rendered', () => {
    expect(() => mount(<Footer />)).not.toThrow()
})
