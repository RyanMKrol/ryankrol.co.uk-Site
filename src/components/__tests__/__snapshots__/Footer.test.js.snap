// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders router only when rendered 1`] = `
<div
  className="Footer"
/>
`;
